%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Friggeri Resume/CV
% XeLaTeX Template
% Version 1.0 (5/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Adrien Friggeri (adrien@friggeri.net)
% https://github.com/afriggeri/CV
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
% Important notes:
% This template needs to be compiled with XeLaTeX and the bibliography, if used,
% needs to be compiled with biber rather than bibtex.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\documentclass[]{friggeri-cv} % Add 'print' as an option into the square bracket to remove colors from this template for printing

\addbibresource{bibliography.bib} % Specify the bibliography file to include publications

\begin{document}

\header{Dr. Philipp}{ Bayer}{Postdoctoral researcher} % Your name and current job title/field

%----------------------------------------------------------------------------------------
%	SIDEBAR SECTION
%----------------------------------------------------------------------------------------

\begin{aside} % In the aside, each new line forces a line break
\section{contact}
\href{mailto:philippbay@gmail.com}{philippbay@gmail.com}
\href{http://github.com/philippbayer}{github.com/philippbayer}
\href{http://twitter.com/philippbayer}{twitter.com/philippbayer}
\section{languages}
German: mother tongue
English: fluent
French \& Japanese: advanced
\section{programming}
Python
Go, Perl, Bash, Java
Ruby on Rails, JavaScript, HTML
\section{research}
Genomics of complex traits in domesticated plants
\section{statistics}
Citations: 951
h-index: 12
i10-index: 13
\end{aside}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\section{education}

\begin{entrylist}
%------------------------------------------------
\entry
{2012--2015}
{PhD {\normalfont Applied Bioinformatics}}
{University of Queensland, Brisbane}
{Working in the Applied Bioinformatics group on the use of genotyping by sequencing to improve the genome assembly of canola.\\Thesis submission date: 23rd September 2015. Date of acceptance: 4th May 2016.}
%------------------------------------------------
\entry
{2010--2012}
{Master {\normalfont of IT}}
{Bond University, Gold Coast}
{Graduated with High Distinction}
%------------------------------------------------
\entry
{2006--2009}
{Bachelor of Science {\normalfont Biology}}
{University of MÃ¼nster, Germany}
{Thesis: Analysis of splicing in two populations of marine plants
using bioinformatic approaches}
\end{entrylist}

\section{employment}
\begin{entrylist}
\entry
{2018-Current}
{Forrest Fellow}
{UWA, Perth}
    {Edwards Lab. Continued work on genomics of complex plants with Forrest Foundation support.}
\entry
{2015--2017}
{Postdoctoral researcher}
{UWA, Perth}
    {Edwards Lab. Researched genetics of complex plants with a focus on canola and wheat. Worked closely with industry partners to improve their breeding programs. Preparing, writing, and publishing research. Currently supervising two interns, co-supervised four PhD students and one MSc student, as well as the local computational infrastructure and data management. Worked extensively on an ARC Industrial Transformation Training Centre application (2018 round).}
\end{entrylist}
%------------------------------------------------

%----------------------------------------------------------------------------------------
%	PUBLICATIONS SECTION
%----------------------------------------------------------------------------------------

\section{publications}
\cite{the_international_wheat_genome_sequencing_consortium_iwgsc_shifting_2018}
\cite{ramirez-gonzalez_transcriptional_2018}
\cite{taylor_indel_2018}
\cite{lee_genomic_2018}
\cite{mousavi_derazmahalleh_adapting_2018}
\cite{mousavi2018exploring}
\cite{hurgobin2017homoeologous}
\cite{kaur2017climate}
\cite{yuan2017runbng}
\cite{yuan2017bionanoanalyst}
\cite{bayer2017assembly}
\cite{yuan2017improvements}
\cite{montenegro2017pangenome}
\cite{kaur2017advanced}
\cite{gacek2016genome}
\cite{golicz2016pangenome}
\cite{hane2017comprehensive}
\cite{barash2016candidate}
\cite{lee2016genome}
\cite{bayer2016genomics}
\vskip 0.3\baselineskip  % have to add these for book chapters - TODO add to friggeri.cls
\cite{visendi2016efficient}
\cite{mason2016centromere}
\cite{bayer2016skim}
\vskip 0.3\baselineskip 
\cite{bayer2015high}
\cite{golicz2015skim}
\vskip 0.3\baselineskip 
\cite{lai2015identification}
\cite{chalhoub2014early}
\cite{mason2014high}
\cite{greshake2014opensnp}
\cite{dattolo2013acclimation}

%printbibsection{article}{articles in peer-reviewed journals} % Print all articles from the bibliography

%\printbibsection{book}{books} % Print all books from the bibliography

%\begin{refsection} % This is a custom heading for those references marked as "inproceedings" but not containing "keyword=france"
%\nocite{*}
%\printbibliography[sorting=chronological, type=inproceedings, title={international peer-reviewed conferences/proceedings}, notkeyword={france}, heading=subbibliography]
%\end{refsection}

%\begin{refsection} % This is a custom heading for those references marked as "inproceedings" and containing "keyword=france"
%\nocite{*}
%\printbibliography[sorting=chronological, type=inproceedings, title={local peer-reviewed conferences/proceedings}, keyword={france}, heading=subbibliography]
%\end{refsection}

%----------------------------------------------------------------------------------------


\newpage
%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------
\section{experience}

\begin{entrylist}
\entry
{2012--Current}
{Research collaboration with Bayer CropScience}
{Ghent, Belgium}
{Continued collaboration with Bayer CropScience on their plant breeding projects which includes several week-long visits to Bayer.}
%------------------------------------------------
\entry
{2011--Current}
{Co-founder openSNP.org}
{Germany/Australia}
{A project for customers of genotyping companies like 23andMe to share their data with scientists around the world, for free. Partially wrote and still maintain the site's Ruby on Rails code-base, interact and manage with the community of 5000 users, administration of the site's servers, and supervision of contributors.}
%------------------------------------------------
\entry
{2013--Current}
{Software Carpentry and Data Carpentry instructor}
{Australia}
{Certified Software Carpentry and Data Carpentry instructor. Software Carpentry is a Mozilla/Alfred P. Sloan Foundation funded non-profit organization which teaches best programming practices (structured programming, reproducible research, version control etc.) in bootcamps to scientists around the world. Data Carpentry is a sister-organisation that focuses on teaching best data management practices.}
%------------------------------------------------
\entry
{2018}
{Research Bazaar Organising committee}
{UWA, Perth}
{ResBaz is a world-wide three-day festival promoting digital literacy. As member of the organising committee I searched for helpers and teachers, drafted the timeplan, designed the web page, raised funding, succeeded in getting government MP to hold keynote}
%------------------------------------------------
\entry
{2017--Current}
{Hacky Hour Founder}
{UWA, Perth}
{Founded the Hacky Hour at UWA, a weekly get-together of researchers and staff working with programming and data, doubles as a help-desk for students with programming problems.}
%------------------------------------------------
\entry
{2017--Current}
{Mozilla Open Science Leadership mentor}
{UWA, Perth}
{Mentoring open source programmers and researchers on how to streamline and grow open source and open science projects under the umbrella of Mozilla.}
%------------------------------------------------
\entry
{2016--Current}
{EMBL-ABR Head of Nodes member, Open Science Special Interest Group member}
{UWA, Perth}
{EMBL-ABR is an Australian-wide network supporting the technical needs of life sciences researchers. Members of the group of Head of Nodes meet monthly to discuss the way forward for the organisation. The Open Science Special Interest Group meets bimonthly to discuss how EMBL-ABR can advance open science in Australia.}
%------------------------------------------------
\entry
{2016--2017}
{COMBINE WA Representative}
{UWA, Perth}
{COMBINE is the student and early career researcher subcommittee of the Australian Bioinformatics and Computational Biology Society (ABACBS). As the local representative I organise or help organise workshops and regular networking events.}
%------------------------------------------------
\end{entrylist}

\newpage
%----------------------------------------------------------------------------------------
%	AWARDS SECTION
%----------------------------------------------------------------------------------------

\section{awards \& funding}

\begin{entrylist}
%------------------------------------------------
\entry
{2018}
{Forrest Research Foundation Non-stipendiary Fellowship}
{UWA}
{Three year fellowship to pursue research at UWA, part of the three inaugural Forrest Fellows}
%------------------------------------------------
\entry
{2017}
{UWA Research Collaboration Award}
{UWA}
{\$28,100 to fund a seagrass microbiome sequencing project}
%------------------------------------------------
\entry
{2014}
{GRDC Travel Award}
{GRDC}
{Travel cost scholarship}
%------------------------------------------------
\entry
{2014}
{SAFS Travel Award}
{University of Queensland}
{Travel cost scholarship}
%------------------------------------------------
\entry
{2013}
{Bayer Grants4Apps}
{Bayer HealthCare}
{Grant to cover openSNP running costs}
%------------------------------------------------
\entry
{2011--2014}
{Two postgraduate scholarships}
{University of Queensland}
{My PhD was supported by two scholarships from UQ for tuition fees and living costs.}
%------------------------------------------------
\entry
{2012}
{First place in PLOS/Mendeley Binary Challenge}
{Won with openSNP.org}
{Won first price in a competition aimed towards the advancement of open science}
%------------------------------------------------
\entry
{2009-2011}
{Master IT}
{Bond University}
{5x Top of class,  3x Vice-Chancellor's List for Academic Excellence, 1x IT Award Academic Excellence. Graduated with High Distinction. Recipient of John Oglethorpe Medal for highest GPA of all IT students graduating that semester}
\end{entrylist}


%----------------------------------------------------------------------------------------
%	TEACHING SECTION
%----------------------------------------------------------------------------------------
\section{teaching}

\begin{entrylist}
\entry
{2018}
{Teaching R}
{Telethon Kids Institute, Perth}
{Introduction to genomics analysis in R}
%------------------------------------------------
\entry
{2018}
{Teaching Data Carpentry}
{Research Bazaar, University of Western Australia, Perth}
{Introduction to genomics and shell. Part of the planning committee.}
%------------------------------------------------
\entry
{2017}
{Teaching Software Carpentry}
{Research Bazaar, Curtin University, Perth}
{Introduction to data manipulation using Python}
%------------------------------------------------
\entry
{2016}
{Teaching and hosting Data Carpentry}
{UWA, Perth}
{Hosted, planned, and set up the first Data Carpentry workshop at UWA, taught best data management practices}
%------------------------------------------------
\entry
{2016--Current}
{University teaching}
{UWA, Perth}
{Co-teach and co-supervise SCIE4002, computational analysis for biology and biomedical MSc students. Set up and maintain the computational infrastructure needed for practicals. In 2017, the course has been judged 'consistenly excellent' over six semesters based on student evaluations.}
%------------------------------------------------
\entry
{2016}
{Teaching Software Carpentry}
{Research Bazaar, Murdoch University, Perth}
{Taught introduction to Python}
%------------------------------------------------
\end{entrylist}
\begin{entrylist}
\entry
{2016}
{Teaching and hosting Software Carpentry}
{UQ, Brisbane}
{Hosted, planned, and set up the first Software Carpentry workshop at UQ. Taught introduction to programming.}
%------------------------------------------------
\entry
{2014}
{Teaching Software Carpentry}
{Melbourne}
{Taught basic to intermediate Python as well as documentation and assisted at bootcamp in Melbourne.}
%------------------------------------------------
\entry
{2013}
{Teaching Software Carpentry}
{Adelaide}
{Assisted Software Carpentry bootcamp in Adelaide}
%------------------------------------------------
\entry
{2009--2011}
{Tutoring}
{Bond University}
{Tutored students in Intro to Programming (Java), Database Management (Oracle/MySQL) and Networks \& Applications, held several all-day refresher courses before exams}
%------------------------------------------------
\end{entrylist}


\section{Presentations}
\begin{entrylist}
%------------------------------------------------
\entry
{2018}
{The path of least resistance (genes) - mining plant genomes for disease resistance}
{}
{COMBINE/Pawsey bioinformatics symposium}
%------------------------------------------------
\entry
{2018}
{Early Career Researcher Panel - What have I learnt at the beginning of my research career?}
{}
{Combined Biological Sciences Meeting 2018}
%------------------------------------------------
\entry
{2018}
{ScienceCafe - STEM outreach aimed at year 10 students}
{}
{UWA}
%------------------------------------------------
\entry
{2017}
{The future of wheat research}
{}
{Wheat showcase, UWA}
%------------------------------------------------
\entry
{2017}
{Skipping the assembly step â what we can learn from looking at sequences directly}
{}
{Pawsey Roadshow, UWA, Perth}
%------------------------------------------------
\entry
{2017}
{The State of Bioinformatics in High Performance Computing in 2017}
{}
{HPCAC Conference, Perth}
%------------------------------------------------
\entry
{2017}
{Towards better plant breeding at UWA}
{}
{COMBINE event, Perth}
%------------------------------------------------
\entry
{2017}
{Improving Plant Breeding using KNetMiner}
{}
{Plant And Animal Genome conference, San Diego}
%------------------------------------------------
\entry
{2016}
{Towards a canola pan-genome: cautionary tales from the assembly bench}
{}
{CCDM, Curtin University}
%------------------------------------------------
\entry
{2016}
{Sharing Experience: What Can We Learn from Each Other Developing Plant Informatics Systems}
{}
{Plant And Animal Genome conference, San Diego}
%------------------------------------------------
\entry
{2015}
{Assessing and validating the amphidiploid genome of \textit{Brassica napus} using genotyping by sequencing}
{}
{Plant And Animal Genome conference, San Diego}
%------------------------------------------------
\entry
{2015}
{Using skim-based genotyping by sequencing for trait association and QTL cloning in Brassica napus}
{}
{Plant And Animal Genome conference, San Diego}
%------------------------------------------------
\entry
{2014}
{Assembling and validating the genome of the \textit{Brassica napus} using skim-based genotyping by sequencing}
{}
{University of Queensland, GenGen Seminar Series}
%------------------------------------------------
\entry
{2012}
{openSNP: Crowdsourcing Genome Wide Association Studies}
{}
{28th Chaos Communication Congress, Berlin}
%------------------------------------------------
\end{entrylist}
\end{document}
